estargz/build.go

```diff
@@ -65,12 +66,21 @@ func WithCompressionLevel(level int) Option {
 	}
 }
 
+// WithPrioritizedFiles option specifies the list of prioritized files.
+// These files must be a complete path relative to "/" (e.g. "foo/bar",
+// "./foo/bar")
+func WithPrioritizedFiles(files []string) Option {
+	return func(o *options) {
```

Comment:
This should verify `files` and return `error`.

Can be a follow-up PR.

Reply:
I'll work on it in the following up PR.